Client/Frontend/Home/Pocket/FxHomePocketViewModel.swift

```diff
@@ -105,17 +105,43 @@ class FxHomePocketViewModel {
 
     // MARK: - Private
 
+    func updatePocketStoryViewModels(with stories: [PocketStory]) {
+        self.pocketStoriesViewModels = []
+        for story in stories {
+            self.bind(pocketStoryViewModel: .init(story: story))
+        }
+    }
+
     private func getPocketSites(completion: @escaping () -> Void) {
         pocketAPI
             .globalFeed(items: UX.numberOfItemsInSection)
             .uponQueue(.main) { [weak self] (pocketStory: [PocketFeedStory]) -> Void in
-            let globalTemp = pocketStory.map(PocketStory.init)
-            self?.pocketStoriesViewModels = []
-            for story in globalTemp {
-                self?.bind(pocketStoryViewModel: .init(story: story))
+                var globalTemp = pocketStory.map(PocketStory.init)
```

Comment:
`()` instead of `.init`?
```suggestion
                var globalTemp = pocketStory.map(PocketStory())
```